cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/AzureClusterSettingsStep.tsx

```diff
@@ -1,5 +1,5 @@
 // React imports
-import React, { useContext, useState } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
```

Comment:
is useEffect being used?!

Reply:
Yes I think after latest merge with top of tree I see it being used on line 108; though previously it was not